workshops/maze/css/style.css

```diff
@@ -0,0 +1,888 @@
+}
+@media (max-width: 480px) {
+  .menubar {
+    padding: 20px;
+    border-bottom: solid 1px #dfe2e7;
+  }
+}
+@media (max-width: 768px) {
+  .menubar {
+    display: none;
+  }
+}
+@media (min-width: 768px) {
+  .content-root {
+    padding-left: 230px;
+  }
+  .menubar {
+    position: absolute;
+    left: 0;
+    top: 0;
+    bottom: 0;
+    width: 230px;
+    border-right: solid 1px #dfe2e7;
+  }
+  .menubar.fixed {
+    position: fixed;
+    overflow-y: auto;
+  }
+  .menubar.fixed {
+    -webkit-overflow-scrolling: touch;
+  }
+  .menubar.fixed::-webkit-scrollbar {
+    width: 15px;
+    height: 15px;
+  }
+  .menubar.fixed::-webkit-scrollbar-thumb {
+    background: #ddd;
+    -webkit-border-radius: 8px;
+    border-radius: 8px;
+    border: solid 4px #fff;
+  }
+  .menubar.fixed:hover::-webkit-scrollbar-thumb {
+    background: #999;
+    -webkit-box-shadow: inset 2px 2px 3px rgba(0,0,0,0.2);
+    box-shadow: inset 2px 2px 3px rgba(0,0,0,0.2);
+  }
+}
+.menubar {
+  font-size: 0.9em;
+}
+.menu ul.level-1 > li + li {
+  margin-top: 20px;
+}
+.menu a {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  position: relative;
+  display: block;
+  padding-top: 1px;
+  padding-bottom: 1px;
+  margin-right: -30px;
+}
+.menu a,
+.menu a:visited {
+  color: #2badad;
+}
+.menu a:hover {
+  color: #228a8a;
+}
+.menu a.level-1 {
+  font-family: montserrat, sans-serif;
+  text-transform: uppercase;
+  font-size: 0.9em;
+  font-weight: bold;
+}
+.menu a.level-1,
+.menu a.level-1:visited {
+  color: #9090aa;
+}
+.menu a.level-1:hover {
+  color: #565666;
+}
+.menu a.level-2 {
+  font-weight: normal;
+}
+.menu a.level-3 {
+  font-weight: normal;
+  font-size: 0.9em;
+  padding-left: 10px;
+}
+.menu a.active {
+  font-weight: bold !important;
+}
+.menu a.active,
+.menu a.active:visited,
+.menu a.active:hover {
+  color: #505050 !important;
+}
+.menu a.active:after {
+  content: '';
+  display: block;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  position: absolute;
+  top: 10px;
+  right: 30px;
+  width: 9px;
+  height: 3px;
+  -webkit-border-radius: 2px;
+  border-radius: 2px;
+  background: #2badad;
+}
+code .string,
+code .number {
+  color: #3ac;
+}
+code .init {
+  color: #383;
+}
+code .keyword {
+  font-weight: bold;
+}
+code .comment {
+  color: #adadcc;
+}
+.large-brief .content > h1:first-child + p,
+.content > p.brief {
+  font-size: 1.3em;
+  font-family: Open Sans, sans-serif;
+  font-weight: 300;
+}
+.title-area {
+  min-height: 100px;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+  text-align: center;
+  border-bottom: solid 1px #dfe2e7;
+  overflow: hidden;
+}
+.title-area > img.bg {
+  z-index: 0;
+  position: absolute;
+  left: -9999px;
+}
+.title-area > div {
+  position: relative;
+  z-index: 1;
+}
```

Comment:
Also, it'd be great if the source `.styl` file was included with this so we have the flexibility to modify this workshop's CSS in the future.